README.md

```diff
@@ -28,15 +28,25 @@ from cptv import CPTVReader
 
 with open(filename, "rb") as f:
     reader = CPTVReader(f)
-    print(reader.device_name)
-    print(reader.timestamp)
-    print(reader.x_resolution)
-    print(reader.y_resolution)
-
-    for frame, t_offset in reader:
-        # Do something with frame.
-        # Each frame is a 2D numpy array.
-
+    print("version:", reader.version)
```

Comment:
This is now duplicated with read.py.   Would a link be  better?

Reply:
Good thinking. Fixed.